core/consensus/common/src/import_queue.rs

```diff
@@ -354,10 +317,10 @@ enum BlockImportError {
 
 /// Import a bunch of blocks.
 fn import_many_blocks<'a, B: BlockT, V: Verifier<B>>(
-	import_handle: &BlockImport<B, Error=ClientError>,
```

Comment:
The main point of this was to avoid the circular deps that arises if you use "Client" in here...